src/bun.js/bindings/BunJSCModule.cpp

```diff
@@ -244,6 +247,62 @@ JSC_DEFINE_HOST_FUNCTION(functionCreateMemoryFootprint, (JSGlobalObject * global
     return JSValue::encode(JSCMemoryFootprint::create(vm, globalObject));
 }
 
+JSC_DECLARE_HOST_FUNCTION(functionNeverInlineFunction);
+JSC_DEFINE_HOST_FUNCTION(functionNeverInlineFunction, (JSGlobalObject * globalObject, CallFrame* callFrame))
+{
+    return JSValue::encode(setNeverInline(globalObject, callFrame));
+}
+
+JSC_DECLARE_HOST_FUNCTION(functionStartSamplingProfiler);
+JSC_DEFINE_HOST_FUNCTION(functionStartSamplingProfiler, (JSC::JSGlobalObject * globalObject, JSC::CallFrame* callFrame))
+{
+    JSC::VM& vm = globalObject->vm();
+    JSC::SamplingProfiler& samplingProfiler = vm.ensureSamplingProfiler(WTF::Stopwatch::create());
+
+    JSC::JSValue directoryValue = callFrame->argument(0);
+    JSC::JSValue sampleValue = callFrame->argument(1);
+
+    auto scope = DECLARE_THROW_SCOPE(vm);
+    if (directoryValue.isString()) {
```

Comment:
I recall `WTF::FileSystem` causing build issues on Ubuntu due to the usage of stdc++fs

but that might have gone away after upgrading to libc++20a

Reply:
but I can tweak this

Reply:
any recommended APIs for FS? I was avoiding a C++ standard based on what I saw in the code.